railties/lib/rails/test_unit/runner.rb

```diff
@@ -42,12 +45,7 @@ def run(argv = [])
         end
 
         def load_tests(argv)
-          patterns = extract_filters(argv)
-
-          tests = Rake::FileList[patterns.any? ? patterns : "test/**/*_test.rb"]
-          tests.exclude("test/system/**/*", "test/dummy/**/*") if patterns.empty?
-
-          tests.to_a.each { |path| require File.expand_path(path) }
+          test_paths(argv).to_a.each { |path| require File.expand_path(path) }
```

Comment:
I don't think the extra layers of indirection buys us much here, we're laying more concrete that it's suggested we want to mold further in another release.

Let's start even simpler and just extract the 2 patterns ala:

```ruby
def load_tests(argv)
  patterns = extract_filters(argv)

  tests = Rake::FileList[patterns.presence || default_test_glob]
  tests.exclude(default_test_exclude_glob) if patterns.empty?

  tests.to_a.each { |path| require File.expand_path(path) }
end

def default_test_glob
  ENV["DEFAULT_TEST"] || "test/**/*_test.rb"
end

def default_test_exclude_glob
  ENV["DEFAULT_TEST_EXCLUDE"] || "test/{system,dummy}/**/*_test.rb"
end
```